_posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc

```diff
@@ -1134,7 +1133,9 @@ infrastructure-live
 === Deploy the ECS Deploy Runner
 
 [.exceptional]
-IMPORTANT: This guide is currently only compatible with ECS deploy runner version v0.23.x
+IMPORTANT: **This guide is currently only compatible with ECS deploy runner version v0.27.x.** We are working on a
+series of updates to the Gruntwork CI/CD pipeline spread out across multiple versions. Once complete, we will update
+this guide to the latest version. In the meantime, use `v0.27.2` for a stable deployment process.
```

Comment:
@josh-padnick what do you think of this alternative wording?

Reply:
Sounds great!